sdk/identity/azure-identity/pom.xml

```diff
@@ -93,6 +93,14 @@
       <artifactId>KeePassJava2</artifactId>
       <version>2.1.4</version> <!-- {x-version-update;org.linguafranca.pwdb:KeePassJava2;external_dependency} -->
     </dependency>
+
+    <!-- This dependency pins the version to 2.4.7 and is required to be pinned as part of issue https://github.com/Azure/azure-sdk-for-java/issues/22807.
+     Don't remove this until the version gets pinned in msal4j library.-->
+    <dependency>
+      <groupId>net.minidev</groupId>
+      <artifactId>json-smart</artifactId>
+      <version>2.4.7</version> <!-- {x-version-update;net.minidev:json-smart;external_dependency} -->
+    </dependency>
```

Comment:
There was PR that had already done this - https://github.com/Azure/azure-sdk-for-java/pull/15793
Then it was removed in PR - https://github.com/Azure/azure-sdk-for-java/pull/21234

The reported GitHub issue is for azure-identity 1.2.3. Is this still an issue with the latest version 1.3.1?

Reply:
Also, the package that brings in the version range for `json-smart` is not used anymore - `com.nimbusds:oauth2-oidc-sdk`

Reply:
Yeah, 
Ideally msal4j should pin this.
While we get msal4j to pin this version, we're pinning it azure-identity.

Reply:
The previously pinned version was affected by severe level CVE as well, so that's why it was removed as well.
But now the latest version has that CVE fixed, so it makes sense to pin to that version.

Reply:
oauth2-oidc-sdk is used by msal4j.